cmd/fleet/serve.go

```diff
@@ -726,7 +726,7 @@ the way that the Fleet server works.
 				}
 			}
 
-			if appCfg.MDM.EnabledAndConfigured {
+			if appCfg.MDM.EnabledAndConfigured || appCfg.MDM.WindowsEnabledAndConfigured {
```

Comment:
this is a pseudo-unrelated fix: it allows you to run the cron if you only have windows MDM turned on.

along with this change, I have also modified the `Reconcile*Profiles` functions in:

- server/service/apple_mdm.go
- server/service/microsoft_mdm.go